docs/research/622-patterns.md

```
@@ -0,0 +1,439 @@
+          code,
+        }),
+      },
+    );
+
+    const { access_token } = await response.json();
+    return Response.redirect(`https://count.racku.la/?token=${access_token}`);
+  },
+};
+```
+
+2. **GitHub App/OAuth App:**
```

markdownlint-cli2 (0.18.1)

Ordered list item prefix
Expected: 1; Actual: 2; Style: 1/2/3

(MD029, ol-prefix)